lib/FieldType/Mapper/RichTextFormMapper.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * This file is part of the eZ RepositoryForms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+
+namespace EzSystems\RepositoryForms\FieldType\Mapper;
+
+use eZ\Publish\Core\FieldType\RichText\Type;
+use EzSystems\RepositoryForms\Data\FieldDefinitionData;
+use EzSystems\RepositoryForms\FieldType\FieldTypeFormMapperInterface;
+use Symfony\Component\Form\FormInterface;
+
+class RichTextFormMapper implements FieldTypeFormMapperInterface
+{
+    public function mapFieldDefinitionForm(FormInterface $fieldDefinitionForm, FieldDefinitionData $data)
+    {
+        $fieldDefinitionForm
+            ->add(
+                'numRows',
+                'integer',
+                [
+                    'empty_data' => 10,
+                    'required' => true,
+                    'property_path' => 'fieldSettings[numRows]',
+                    'label' => 'field_definition.ezrichtext.num_rows',
+                ]
+            )
+            ->add(
+                'tagPreset',
```

Comment:
That setting is really a leftover from the time when RichText was improved XmlText field type.
It does not go deeper than UI, and as far as I'm concerned we can remove it. But maybe we should still ping PM about it.